src/style.css

```diff
@@ -1 +1,56 @@
-/* add styles here */
+/* #region page */
+html {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). If you need to reset default margins and paddings, consider using a CSS reset file or a more specific class.